tools/config/__init__.py

```diff
@@ -1354,10 +1354,10 @@ def get_features(self):
         self.cumulative_overrides['features']\
             .update_target(self.target)
 
-        for feature in self.target.features:
+        for feature in list(self.target.features):
             if feature not in ALLOWED_FEATURES:
-                raise ConfigException(
-                    "Feature '%s' is not a supported features" % feature)
+                print("[WARNING] Feature '%s' is not a supported feature" % feature)
+                self.target.features.remove(feature)
```

Comment:
Removing this feature will prevent that feature from being defined or having any effect, which is a nice guarantee.